erica/erica_legacy/api/v1/endpoints/tax.py

```diff
@@ -3,7 +3,7 @@
 from fastapi import HTTPException, status, APIRouter
 from starlette.responses import FileResponse
 
-from erica.application.tax_number_validation.check_tax_number_dto import CheckTaxNumberDto, StateAbbreviation
+from erica.domain.tax_number_validation.check_tax_number import StateAbbreviation, CheckTaxNumberPayload
```

Comment:
In fact the presentationlayer should not access the domainlayer - use DTOs. (it could be that the DTO and the model for the payload have the same fields)

Reply:
This is v1 of the application. I added the CheckTaxNumberPayload because I changed the CheckTaxNumberRequestProcessor for the corresponding service. We could add a second dto, which is equal to the Payload model. However, as this is v1, I think it might be fine like that.

Reply:
Yes, I saw that this is v1 and therefore will be deleted. I just wanted to draw attention to that :) .